FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.3.1
+- [removed] Client id usage in api call and respective checks in the code.
```

Comment:
s/removed/changed/ since nothing is removed from a client perspecitve.

Reply:
done